replayer-fixtures/2cb72062a815e7fe93ad97900474da9e

```diff
@@ -0,0 +1 @@
+{"member":{"id":10100487,"email":"vincent@sumofus.org","first_name":"Example","last_name":"Member","postal":"1000","country":"US"}}
```

Comment:
Your fixture body has the member hash (including the email) but doesn't have the top level email key. I thought that was the way you modified the API?